lib/buildFactory.js

```diff
@@ -265,6 +269,16 @@ class BuildFactory extends BaseFactory {
                     ];
 
                     modelConfig.templateId = job.templateId;
+                    if (job.templateId !== undefined && job.templateId !== null) {
+                        Promise.all([
+                            templateFactory.get({ id: job.templateId }),
+                            pipelineFactory.get({ id: job.pipelineId })
+                        ]).then(([t, p]) =>
+                            logger.info(
+                                `pipelineId:${p.id}: ${p.name} uses template ${t.namespace}/${t.name}@${t.version}`
```

Comment:
can we store this in build model ? We have lot's of users asking to display template information in build detail page.

Reply:
The build model already has a [`templateId`](https://github.com/screwdriver-cd/data-schema/blob/1cc4a369f3b77c6ea2090ae8fbc1a0fed428ac51/models/build.js#L131-L134). Therefore, it is possible to display template informations without saving it in the DB, and the flexibility is increased.

Reply:
can this is done based on log level ? https://github.com/screwdriver-cd/logger/blob/master/index.js#L6

This is `logger.info`, and logger will drop if log level > `info` and making 2 DB calls (already excessive) will be a waste in this situation.